packages/drawer/src/react/__stories__/index.story.js

```diff
@@ -143,13 +143,14 @@ storiesOf('drawer', module)
     <Drawer
       base={
         <Row
-          image={<Row.Image src="https://cataas.com/cat" />}
-          title="Look at me! I'm a <Row />!"
-          metadata1={['Kitten McCatbuns', '23 hours of cuteness']}
-          actionBarVisible
           actionBar={[
             <Row.Action key="iHeartCats" icon={<Icon id="more" />} />
           ]}
+          actionBarVisible
+          image={<Row.Image src="https://cataas.com/cat" />}
+          metadata1={['Kitten McCatbuns', '23 hours of cuteness']}
+          size={Row.sizes.medium}
```

Comment:
just curious, why was this necessary?

Reply:
it keeps the snapshots from changing - since medium was the previous default behaviour